srcpkgs/pulseeffects/template

```diff
@@ -1,20 +1,24 @@
 # Template file for 'pulseeffects'
 pkgname=pulseeffects
-version=5.0.3
+version=5.0.4
 revision=1
 build_style=meson
-hostmakedepends="itstool pkg-config gettext"
+hostmakedepends="itstool pkgconf gettext"
 makedepends="boost-devel glib-devel gsettings-desktop-schemas-devel
  gst-plugins-bad1-devel gtkmm-devel libebur128-devel lilv-devel
- pipewire-devel python3-gobject-devel sratom-devel zita-convolver-devel
- libsndfile-devel libbs2b-devel"
-depends="calf gsettings-desktop-schemas gst-plugins-bad1
- gst-plugins-good1 pipewire python3-gobject python3-scipy zita-convolver
- gstreamer1-pipewire"
+ pipewire-devel sratom-devel zita-convolver-devel
+ rubberband-devel libsamplerate-devel libsndfile-devel"
+depends="calf gstreamer1-pipewire gst-plugins-good1"
 short_desc="Sound effects for systems using PipeWire"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/wwmm/pulseeffects"
+changelog="https://github.com/wwmm/pulseeffects/blob/v${version}/CHANGELOG.md"
 distfiles="https://github.com/wwmm/pulseeffects/archive/v${version}.tar.gz"
-checksum=2e14858918b54bee5f6e4898cc803ae2170b4d624407fef39e0831b6584c4a4f
-python_version=3
+checksum=3fa482e2261fe467e30c05e85a55904806b0dc141834fe8b5cfeffedb6da65e8
+conflicts="pulseeffects-legacy"
```

Comment:
Why?

Reply:
Because `pulseeffects-legacy` already has `conflicts="pulseeffects"`.
And both packages have many common files. Though I guess it's possible to rename them to change `pulseeffects` to `pulseeffects-legacy`.

Reply:
Adding another conflict does not change anything, it just results in two error messages instead of one.

Reply:
@Duncaen so `conflicts`works two-way? Interesting, didn't know that. Just tested it and it works. 

P. S. Out of curiosity I'm trying to find out which part of `xbps` is responsible for handling conflicts two-way, but reading this file didn't help much: https://github.com/void-linux/xbps/blob/master/lib/transaction_check_conflicts.c
I would appreciate explaining how does it work.

Reply:
There is no need to do any "two-way" conflict checks as each package that is installed and has `conflicts=` defined will be checked against installed or to be installed packages.